packages/common/src/operations.graphql

```diff
@@ -317,6 +317,59 @@ mutation updateLocation($input: UpdateLocationInput!) {
   }
 }
 
+mutation deleteLocation($storeId: String!, $input: DeleteLocationInput!) {
+  deleteLocation(storeId: $storeId, input: $input) {
+    ... on DeleteLocationError {
+      __typename
+      error {
+        description
+        ... on RecordNotFound {
+          __typename
+          description
+        }
+        ... on DatabaseError {
+          __typename
+          description
+          fullError
+        }
+        ... on RecordBelongsToAnotherStore {
+          __typename
+          description
+        }
+        ... on LocationInUse {
+          __typename
+          description
+          stockLines {
+            ... on StockLineConnector {
+              __typename
```

Comment:
am returning a little bit of the stock lines and invoice lines though I doubt that they are useful